packages/server-wallet/e2e-test/README.md

```diff
@@ -11,35 +11,35 @@ Channel | |    | |              Channel | |    | |
         | v    | v                      | v    | v
       +-+-+----+-+--+                 +-+-+----+-+--+
       |             +-------2-------->+             |
-      | Ping Client |  POST /inbox    | Pong Client |
+      | Payer Client |  POST /inbox    | Receiver Client |
```

Comment:
I don't know how these diagrams were generated, but now they're misaligned :P

Reply:
Manually fixed the diagram for now